be/src/exprs/agg/aggregate_factory.cpp

```diff
@@ -807,9 +809,22 @@ const AggregateFunction* get_aggregate_function(const std::string& name, Primiti
         return AggregateFuncResolver::instance()->get_aggregate_info(func_name, arg_type, return_type, is_null);
     } else if (binary_type == TFunctionBinaryType::SRJAR) {
 #ifdef STARROCKS_WITH_HDFS
-        return newJavaUDAFFunction(is_null);
+        return getJavaUDAFFunction(is_null);
 #endif
     }
     return nullptr;
 }
+
+const AggregateFunction* get_window_function(const std::string& name, PrimitiveType arg_type, PrimitiveType return_type,
+                                             bool is_null, TFunctionBinaryType::type binary_type) {
+    if (binary_type == TFunctionBinaryType::BUILTIN) {
+        return AggregateFuncResolver::instance()->get_aggregate_info(name, arg_type, return_type, is_null);
+    } else if (binary_type == TFunctionBinaryType::SRJAR) {
+#ifdef STARROCKS_WITH_HDFS
+        return getJavaWindowFunction();
```

Comment:
Why window function needn't handle `is_null `

Reply:
The window function now assumes that the input is always nullable